code/modules/mob/living/simple_animal/abnormality/he/white_lake.dm

```diff
@@ -123,16 +127,15 @@
 	if (slot != ITEM_SLOT_HANDS)
 		REMOVE_TRAIT(src, TRAIT_NODROP, src)
 
-/obj/item/ego_weapon/flower_waltz/pre_attack(atom/A, mob/living/user, params)
+/obj/item/ego_weapon/flower_waltz/attack(mob/living/target, mob/living/carbon/human/user, proximity)
+	var/enemy = 0
+	for(var/found_faction in target.faction)
+		if(found_faction in nemesis_factions) // if we are hitting a nemesis...
+			force += faction_bonus_force
+			enemy = 1
```

Comment:
```suggestion
			enemy = 1
			break
```
This will stop it from stacking this on a mob that has multiple instances of `"hostile" faction... for whatever reason.

Reply:
i understand make some failsafes, but this really shouldnt be necessary. Change made anyways